content/docs/testing-recipes.md

```diff
@@ -228,11 +232,12 @@ it("renders user data", async () => {
 
 ---
 
-### Mocking Modules {#mocking-modules}
+### (Mocking Modules) تقليد الوحدات  {#mocking-modules}
```

Comment:
```suggestion
### (Mocking Modules) محاكاة الوحدات  {#mocking-modules}
```

Reply:
I gonna work on them, I was trying as much as I could to not translate in Egyptian way, I'll apply ur suggestions!